source/style-guide/includes/rendered-icon-table.rst

```diff
@@ -8852,8 +8848,4 @@
    * - ``mms-icon-wrench`` 
      - ``:icon-mms:`wrench``` 
      - :icon-mms:`wrench`
-                
-   * - ``mms-icon-wrench-exclamation`` 
```

Comment:
Question:
Is this icon invalid? I just wanted to make sure that removing it was intentional.

Reply:
I think removing is good if it doesn't exist. I just wanted to confirm that it wasn't an accidental deletion.